src/cocaine-app/yt_worker.py

```diff
@@ -0,0 +1,170 @@
+from logging import getLogger
+from yql.api.v1.client import YqlClient
+import time
+import datetime
+
+
+logger = getLogger('mm.planner')
+
+
+class YqlWrapper(object):
+    def __init__(self, cluster, token, attempts, delay):
+        self._cluster = cluster
+        self._token = token
+        self._attempts = attempts
+        self._delay = delay
+
+    def send_request(self, query, timeout=None):
+        """
+        Send request to YQL: ping verification + attempts retry
+        @return result dict on success, None otherwise
+        """
+        logger.debug('Send YQL request %s', query)
+        result = None
+
+        with YqlClient(db=self._cluster, token=self._token) as yql:
+            try:
+                if not yql.ping():
+                    logger.error("YQL ping failed %s %s", str(self._cluster), str(self._token))
+                    return None
+            except:
+                logger.exception("YQL ping excepted %s %s", str(self._cluster), str(self._token))
+                return None
```

Comment:
Fixed in https://github.com/yandex/mastermind/pull/311/commits/808a99a19c77cd1a34d2ad6416390a670b6394eb